chore: Delete Algolia documentation files and add a simple reference README.md (#142)
algolia/algoliasearch-sfcc-b2c: README.md

# Algolia Search & Discovery for Salesforce B2C Commerce

Algolia's cartridge unlocks best-in-class Search & Discovery on Salesforce B2C Commerce.

It can be used for synchronization of product (incl. price and inventory information), category and content data, and includes native storefront cartridges to power Search & Discovery Experiences on both [SFRA (Storefront Reference Architecture)](https://developer.salesforce.com/docs/commerce/sfra/guide/sfra-overview.html) and SiteGenesis.
It also is compatible with Headless/Composable storefront integrations, such as the [PWA Kit](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/getting-started.html), thanks to Algolia's developer-friendly [InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/), [Autocomplete](https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete) and [Recommend](https://www.algolia.com/doc/ui-libraries/recommend/introduction/getting-started/) UI libraries.

> The integration can be used in conjunction with Algolia’s rich suite of Search & Discovery products such as its user-friendly [Merchandising Studio](https://www.algolia.com/doc/guides/managing-results/rules/merchandising-and-promoting/) and [Search Analytics](https://www.algolia.com/doc/guides/search-analytics/overview/), powerful [AI Search](https://www.algolia.com/doc/guides/getting-started/neuralsearch), [Personalization](https://www.algolia.com/doc/guides/personalization/what-is-personalization/) and [Recommend](https://www.algolia.com/doc/guides/getting-started/neuralsearch/).

Reviewer: Missed this one
```suggestion
The integration can be used in conjunction with Algolia’s rich suite of Search & Discovery products such as its user-friendly [Merchandising Studio](https://www.algolia.com/doc/guides/managing-results/rules/merchandising-and-promoting/) and [Search Analytics](https://www.algolia.com/doc/guides/search-analytics/overview/), powerful [AI Search](https://www.algolia.com/doc/guides/getting-started/neuralsearch), [Personalization](https://www.algolia.com/doc/guides/personalization/what-is-personalization/) and [Recommend](https://www.algolia.com/doc/guides/algolia-recommend/overview/).
```

# Algolia Search & Discovery for Salesforce B2C Commerce

> Algolia’s cartridge unlocks best-in-class Search & Discovery on Salesforce B2C Commerce.

Reviewer: ```suggestion
Algolia's cartridge unlocks best-in-class Search & Discovery on Salesforce B2C Commerce.
```

> It can be used for synchronization of product (incl. price and inventory information), category and content data, and includes native storefront cartridges to power Search & Discovery Experiences on both [SFRA (Storefront Reference Architecture)](https://developer.salesforce.com/docs/commerce/sfra/guide/sfra-overview.html) and SiteGenesis. It also is compatible with Headless/Composable storefront integrations, incl. via the [PWA Kit](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/getting-started.html), accelerated by Algolia’s developer-friendly [InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/), [Autocomplete](https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete) and [Recommend](https://www.algolia.com/doc/ui-libraries/recommend/introduction/getting-started/?client=React) UI libraries. The integration can be used in conjunction with Algolia’s rich suite of Search & Discovery products such as its user-friendly [Merchandising Studio](https://www.algolia.com/doc/guides/managing-results/rules/merchandising-and-promoting/) and [Search Analytics](https://www.algolia.com/doc/guides/search-analytics/overview/), powerful [AI Search](https://www.algolia.com/doc/guides/getting-started/neuralsearch), (Personalization)[https://www.algolia.com/doc/guides/getting-started/neuralsearch/] and [Recommend](https://www.algolia.com/doc/guides/getting-started/neuralsearch/).

Reviewer: ```suggestion
It can be used for synchronization of product (incl. price and inventory information), category and content data, and includes native storefront cartridges to power Search & Discovery Experiences on both [SFRA (Storefront Reference Architecture)](https://developer.salesforce.com/docs/commerce/sfra/guide/sfra-overview.html) and SiteGenesis.
It also is compatible with Headless/Composable storefront integrations, such as the [PWA Kit](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/getting-started.html), thanks to Algolia's developer-friendly [InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/), [Autocomplete](https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete) and [Recommend](https://www.algolia.com/doc/ui-libraries/recommend/introduction/getting-started/) UI libraries.
The integration can be used in conjunction with Algolia’s rich suite of Search & Discovery products such as its user-friendly [Merchandising Studio](https://www.algolia.com/doc/guides/managing-results/rules/merchandising-and-promoting/) and [Search Analytics](https://www.algolia.com/doc/guides/search-analytics/overview/), powerful [AI Search](https://www.algolia.com/doc/guides/getting-started/neuralsearch), [Personalization](https://www.algolia.com/doc/guides/personalization/what-is-personalization/) and [Recommend](https://www.algolia.com/doc/guides/getting-started/neuralsearch/).
```

> To learn more about the cartridge and how to get started, check out our documentation for more information.

Reviewer: ```suggestion
To learn more about the cartridge and how to get started, check out [our documentation](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/getting-started/introduction/).
```

## References
- [Algolia Docs](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/getting-started/introduction/)
- [AppExchange Listing](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N4V00000IkzoAUAR&tab=e)
- [Academy: Algolia's Salesforce B2C Commerce Integration](https://academy.algolia.com/training/718dcbf0-786f-11ec-a21b-02d47b69d3fd/overview)
- [General information](https://www.algolia.com/search-solutions/salesforce-commerce-cloud/)

## Need Help?

Here is some helpful documentation that might answer some of your questions:

> - FAQs

Reviewer 2: Does this still contain the corresponding links for each of these?

Reviewer: I'm ok to remove the section entirely, as the documentation has a proper sidebar with all the sections.

- Monitoring
- Algolia Support Articles
- Indexing
- Frontend
- Events

> For feedback, bug reporting, or unresolved issues with the cartridge, please contact us via our [support portal](https://support.algolia.com/hc/en-us/requests/new) or email [support@algolia.com](mailto:support@algolia.com). Please include your cartridge version, Salesforce B2C version, application ID, steps to reproduce your issue and some indication of the severity. Add additional information like error messages, downloaded log files, screenshots, screencasts or details about customizations to help our team better troubleshoot your issues.

Reviewer: ```suggestion
For feedback, bug reporting, or unresolved issues with the cartridge, please contact us via our [support portal](https://support.algolia.com/hc/en-us/requests/new).
Include your cartridge version, Salesforce B2C version, application ID, steps to reproduce your issue and some indication of the severity.
Add additional information like error messages, log files, screenshots, screencasts or details about customizations to help our team better troubleshoot your issues.
```

Reviewer 2: Re removing the specific links in the docs - the idea was that some folks might not notice the most important sections to self-serve their issues/questions like the FAQs in there (and this kind of makes them more visible upfront). But I'm also okay either way.

Reviewer: Sure, no strong opinion, if we keep them links must not be removed
